.env.example

```
@@ -1 +1,13 @@
 DATABASE_URL=
+
+# Better Auth
+BETTER_AUTH_SECRET=p2kG47YycBETeqvinx4wt6NQ50XCAk7m
+BETTER_AUTH_URL=http://localhost:3000 # Base URL of your app
+
+# Cal.com OAuth 2.0 credentials
+# https://cal.com/help/apps-and-integrations/oauth
+CAL_COM_CLIENT_ID=
+CAL_COM_CLIENT_SECRET=
+
+NEXT_PUBLIC_APP_URL=http://localhost:3000 # Base URL of your app
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)